content/feedId.ttl

```diff
@@ -2,8 +2,14 @@
 @prefix skos: <http://www.w3.org/2004/02/skos/core#> .
 :feedId
     skos:prefLabel "feed id"@en;
+    skos:related "versionedFeedId";
     skos:definition '''
-      The id of feed.
-      It is constructed as `{{feed-type}}/{{feed-name}}`.
-      It can be extended to a versioned feed id.
+      The unique id of feed.
```

Comment:
Is there a non-unique one? 
My first attempt is also bad.

We going upper case or lower for "id"?

Reply:
@Christian-MK You can keep comments threaded. 

Whoever we follow for a style guide, it should be decided, documented and easy to follow. This ttl def has both upper and lower case